src/chrome/content/rules/Vip-mobile.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="Vip mobile">
+
+	<target host="vipmobile.rs" />
+	<target host="www.vipmobile.rs" />
+	<target host="automanager.vipmobile.rs" />
+	<target host="asmp.vipmobile.rs" />
+	<target host="cdn.vipmobile.rs" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586562993#L326
Remove cdn to comment.